<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationControlWidget</class>
 <widget class="QDockWidget" name="SimulationControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation Control</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>LaserScanner</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="mBtnLaserScannerCreate">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="mBtnLaserScannerDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="mBtnLaserScannerSaveConfiguration">
         <property name="text">
          <string>Save Configuration</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Scanner to edit:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinBoxScannerIndex">
           <property name="maximum">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Properties</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Range</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxPropertiesRange">
              <property name="suffix">
               <string>m</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Speed</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mSpinBoxPropertiesSpeed">
              <property name="suffix">
               <string>rpm</string>
              </property>
              <property name="minimum">
               <number>60</number>
              </property>
              <property name="maximum">
               <number>3000</number>
              </property>
              <property name="value">
               <number>2400</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Angle start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mSpinBoxPropertiesAngleStart">
              <property name="suffix">
               <string>째</string>
              </property>
              <property name="maximum">
               <number>180</number>
              </property>
              <property name="value">
               <number>45</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Angle stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mSpinBoxPropertiesAngleStop">
              <property name="suffix">
               <string>째</string>
              </property>
              <property name="minimum">
               <number>180</number>
              </property>
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="value">
               <number>315</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Angle step</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxPropertiesAngleStep">
              <property name="suffix">
               <string>째</string>
              </property>
              <property name="minimum">
               <double>0.100000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.050000000000000</double>
              </property>
              <property name="value">
               <double>0.250000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxScannerPositionX">
              <property name="suffix">
               <string>m</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Y</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxScannerPositionY">
              <property name="suffix">
               <string>m</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Z</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxScannerPositionZ">
              <property name="suffix">
               <string>m</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Rotation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Pitch</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxScannerRotationPitch">
              <property name="suffix">
               <string>째</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Roll</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxScannerRotationRoll">
              <property name="suffix">
               <string>째</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Yaw</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mSpinBoxScannerRotationYaw">
              <property name="suffix">
               <string>째</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>-180.000000000000000</double>
              </property>
              <property name="maximum">
               <double>180.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Time</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Factor</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="mSpinBoxTimeFactor">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="mBtnStart">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="mBtnPause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>172</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
